### ISSUE TITLE
Add step to start redis-server

### ISSUE THREAD

**Issue by williamn** [marketplacekit/marketplacekit#36]
When I run `db:seed` I got redis error

```
➜  marketplacekit (master) ✗ php artisan db:seed
Seeding: UsersTableSeeder
Seeding: RolesAndPermissionsSeeder

   Predis\Connection\ConnectionException  : Connection refused [tcp://127.0.0.1:6379]
```

There should be a step to install/start redis-server before running `db:seed`. Or if redis is optional, remove redis configuration value from the `.env` file.

**Comment by david-saint** [comment]
Exactly! but this link would guide on installing and configuring redis on **ubuntu** https://www.digitalocean.com/community/tutorials/how-to-install-and-secure-redis-on-ubuntu-18-04
**windows** - https://www.youtube.com/watch?v=ncFhlv-gBXQ

**Comment by bishweswartheaquarious** [comment]
getting error 
![error](https://user-images.githubusercontent.com/29772900/54816461-45c9ec80-4cba-11e9-9c5a-990c8a06735c.png)

Can anyone help me.

**Comment by geoker** [comment]
So, what is the solution to this predis issue?

**Comment by asharekh** [comment]
is there a way to resolve this issue?
